src/defs.h

```diff
@@ -48,13 +48,22 @@
 /* This macro will be automatically defined at shecc run-time. */
 #ifdef __SHECC__
 #define UNUSED(x) (x)
+#define HOST_PTR_SIZE 4
 #else
 /* suppress GCC/Clang warnings */
 #define UNUSED(x) (void) (x)
+/* configure host data model when using `memcpy()` */
+#define HOST_PTR_SIZE 8
```

Comment:
Is better to use `UINT_MAX` (defined in `<stdint.h>`) instead?

Reply:
> Is better to use `UINT_MAX` (defined in `<stdint.h>`) instead?

We need to configure it according to the host data model. Is it better to use `__SIZEOF_POINTER__` predefined macro?

Reply:
For stage-0 bootstrapping, any C99-compatible compilers should be able to compile shecc from scratch. I prefer the macros defined in `<stdint.h>`.

Reply:
> Is better to use `UINT_MAX` (defined in `<stdint.h>`) instead?

The `UINT_MAX` seems not to be used for getting the platform's pointer size.